mne/coreg.py

```diff
@@ -64,6 +69,46 @@ def _find_high_res_head(subject, subjects_dir):
             return path
 
 
+def _fiducial_coords(points, coord_frame):
+    "Helper function to generate 3x3 array of fiducial coordinates"
+    points_ = dict((p['ident'], p) for p in points if
+                   p['kind'] == FIFF.FIFFV_POINT_CARDINAL and
+                   p['coord_frame'] == coord_frame)
+    return np.array([points_[i]['r'] for i in FIDUCIAL_ORDER])
+
+
+def coregister_fiducials(info, fiducials, tol=0.01):
+    """Create a head-MRI transform by aligning 3 fiducial points
+
+    Parameters
+    ----------
+    info : Info
+        Measurement info object with fiducials in head coordinate space.
+    fiducials : str | list of dict
+        Fiducials in MRI coordinate space (either path to a ``*-fiducials.fif``
+        file or list of fiducials as returned by :func:`read_fiducials`.
+
+    Returns
+    -------
+    trans : Transform
+        The device-MRI transform.
+    """
+    if isinstance(info, string_types):
+        info = read_info(info)
+    if isinstance(fiducials, string_types):
+        fiducials, coord_frame_to = read_fiducials(fiducials)
+    else:
+        coord_frame_to = FIFF.FIFFV_COORD_MRI
+    coords_from, frames_from = _fiducial_coords(info['dig'])
+    if len(set(frames_from)) > 1:
+        raise ValueError("info contains fiducials from different coordinate "
+                         "frames")
+    coord_frame_from = frames_from[0]
+    coords_to = _fiducial_coords(fiducials, coord_frame_to)
+    trans = fit_matched_points(coords_from, coords_to, tol=tol)
```

Comment:
I think this probably isn't the correct function. From fiducials you get a center point and axis directions. Using that geometry analytically is better than using a generic fitting function